jobs/rotate-node.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: batch/v1
```

Comment:
This job has a lot of duplication with the rotate node cronjob. Is it sensible to have the cronjob actually trigger one of these jobs itself? Will definitely save the duplication, and then the rotate node cron really only needs to have the ability to create jobs on the api, and this file can own the node-manager service account and whatnot.

Reply:
This can be considered in a future PR or something. The duplication isn't great, but I'd rather have this merged in than float around until I can get around to generalizing.